01-switzerland-more-involved.ipynb

```diff
@@ -0,0 +1,284 @@
+{
```

Comment:
<p>Kann ihc nix lese</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gabrieltorresgamez/fhnw-san-project/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>this are remnants of tobis branch, which I merged. Should I delete this file?</p>